scripts/updatebreaches.js

```diff
@@ -0,0 +1,97 @@
+"use strict";
+
+const got = require("got");
+
+const AppConstants = require("../app-constants");
+const HIBP = require("../hibp");
+
+
+const FX_RS_RECORDS = `${AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-workspace/collections/fxmonitor-breaches/records`;
+const FX_RS_COLLECTION = `${AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-workspace/collections/fxmonitor-breaches`;
+const FX_RS_PREVIEW_COLLECTION = `${AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER}/buckets/main-preview/collections/fxmonitor-breaches`;
+const FX_RS_USERNAME = AppConstants.FX_REMOTE_SETTINGS_USERNAME;
+const FX_RS_PASSWORD = AppConstants.FX_REMOTE_SETTINGS_PASSWORD;
+
+
+if (!FX_RS_USERNAME || !FX_RS_PASSWORD) {
+  console.error("updatebreaches requires FX_REMOTE_SETTINGS_USERNAME and FX_REMOTE_SETTINGS_PASSWORD.");
+  process.exitCode = 1;
+  return;
+}
+
+
+function whichBreachesAreNotInRemoteSettingsYet(remoteSettingsBreachesSet, hibpBreaches) {
+  const newBreaches = [];
+  for (const breach of hibpBreaches) {
+    if (breach.IsSpamList || breach.IsRetired || !breach.IsVerified || !breach.Domain) {
```

Comment:
Do we need to check `breach.IsFabricated`, per:

https://github.com/mozilla/blurts-server/blob/39e81fecfd01ff8b57aa2b3005b00b2a9aea7272/hibp.js#L143-L151

I'd say bonus points if we could reuse that filter function instead of duplicating the logic in 2+ places, but I agree w/ @nhnt11 that this may be better suited for it's own repo so that you don't have to clone+install all the server dependencies just to run this one script.

Reply:
Good call!